style.css

```diff
@@ -1,166 +1,137 @@
-/* HEADER SECTION */
+/* --GENERAL SHAPE -- */
+body {
+  margin: 0;
+  width: 100%;
+}
 
```

Comment:
[Optional] TO make the page look better and friendly to the user, I suggest you to add some hover, focus on buttons as shown in the figma design:  
<img width="336" alt="image" src="https://user-images.githubusercontent.com/88894525/236651225-95d037e3-1a4b-4ff0-b232-2fcd3db17371.png">